app/src/main/java/com/eero/andevcondemo/MainActivity.java

```diff
@@ -52,7 +52,8 @@ public void afterTextChanged(Editable editable) {
             history.add(new Pair<Integer, Integer>(input, output));
             String textOutput = String.valueOf(output);
             outputView.setText(String.valueOf(output));
-            
+
+            ArrayList<String> test = new ArrayList<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "test" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "test". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)